src/subdomains/supporting/payment/services/transaction-helper.ts

```diff
@@ -428,18 +428,17 @@ export class TransactionHelper implements OnModuleInit {
     paymentMethodOut: PaymentMethod,
     user: User,
   ): Promise<{ kycLimit: number; defaultLimit: number }> {
+    const volume24h =
+      user?.userData.kycLevel < KycLevel.LEVEL_50
```

Comment:
Is `user` optional? 🤔  In the parameter above it is not optional, but maybe the userData relation could be missing. I would add a `?` behind userData or?

Reply:
True, `user` should be optional.